python/acronym/acronym.py

```diff
@@ -0,0 +1,13 @@
+def abbreviate(words):
+    words = words.upper()
```

Comment:
It would be more natural to call upper on the acronym you are returning, rather than capitalizing everything here